src/ProjectTemplates/GeneratedContent.targets

```diff
@@ -22,24 +22,24 @@
       -->
     <PropertyGroup>
       <TemplatePackageVersion_MicrosoftExtensionsAI>9.7.1</TemplatePackageVersion_MicrosoftExtensionsAI>
-      <TemplatePackageVersion_MicrosoftExtensionsAI_Preview>$(PackageVersion)</TemplatePackageVersion_MicrosoftExtensionsAI_Preview>
-      <TemplatePackageVersion_MicrosoftExtensionsHttpResilience>9.7.1</TemplatePackageVersion_MicrosoftExtensionsHttpResilience>
+      <TemplatePackageVersion_MicrosoftExtensionsAI_Preview>9.7.1-preview.1.25365.4</TemplatePackageVersion_MicrosoftExtensionsAI_Preview>
+      <TemplatePackageVersion_MicrosoftExtensionsHttpResilience>9.7.0</TemplatePackageVersion_MicrosoftExtensionsHttpResilience>
```

Comment:
Not sure these changes are right. I'm assuming what you want to ship in 9.7.2 is the templates, and if so, I'd expect this to be:

```xml
<TemplatePackageVersion_MicrosoftExtensionsAI>9.7.2</TemplatePackageVersion_MicrosoftExtensionsAI>
      <TemplatePackageVersion_MicrosoftExtensionsAI_Preview>$(PackageVersion)</TemplatePackageVersion_MicrosoftExtensionsAI_Preview>
      <TemplatePackageVersion_MicrosoftExtensionsHttpResilience>9.7.0</TemplatePackageVersion_MicrosoftExtensionsHttpResilience>
```

Reply:
We're only shipping the template package at 9.7.2. These versions are what the template references in the generated projects, and we need to pin to the already-published versions.